docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,149 @@
+
+**Example**
+
+The following *artifact.xml* file is produced after running the example command shown above. Note that [`Castle.Core`](https://www.nuget.org/packages/Castle.Core/) is a depdenency of `Moq`, so it's included automatically and appears in the *artifacts.xml* manifest file.
+
+```xml
+<StoreArtifacts>
+  <Package Id="newtonsoft.json" Version="10.0.3" />
+  <Package Id="castle.core" Version="4.1.0" />
+  <Package Id="moq" Version="4.7.63" />
+</StoreArtifacts>
+```
+
+## Publishing an app against a target manifest
+
+If you have a target manifest file on disk, you specify the path to the file when publishing your app with the [`dotnet publish`](../tools/dotnet-publish.md) command:
+
+```console
+dotnet publish --manifest <PATH_TO_MANIFEST_FILE>
+```
+
+**Example**
+
+```console
+dotnet publish --manifest manifest.xml
+```
+
+You're required to deploy the resulting published app to an environment that has the packages described in the target manifest. Failing to do so results in the app failing to start.
+
+Specify multiple target manifests when publishing an app by repeating the option and path (for example, `--manifest manifest1.xml --manifest manifest2.xml`). When you do so, the app is trimmed for the union of packages specified in the target manifest files provided to the command.
+
+## Specifying target manifests in the project file
+
+Instead of specifying target manifests with the [`dotnet publish`](../tools/dotnet-publish.md) command, you can specify them in the project file as a semicolon-separated list of paths under a **\<TargetManifestFiles>** tag.
+
+```xml
+<PropertyGroup>
+  <TargetManifestFiles>manifest1.xml;manifest2.xml</TargetManifestFiles>
+</PropertyGroup>
+```
+
+Specify the target manifests in the project file only when the target environment for the app is well-known, such as for .NET Core projects. This isn't the case for open-source projects. The users of an open-source project likely deploy to different production environments, which probably have different sets of packages pre-installed. Maintainers of the project can't make assumptions about the target manifest in such open-source environments, so developers should instead rely on the `--manifest` option of [`dotnet publish`](../tools/dotnet-publish.md).
+
+## ASP.NET Core implicit store
+
+The runtime package store feature is used implicitly by ASP.NET Core apps. The targets in [`Microsoft.NET.Sdk.Web`](https://github.com/aspnet/websdk) include manifests referencing the implicit package store that's available when the .NET Core SDK is installed on the target system. Running `dotnet publish` for an app results in a published app that contains only the app and its assets and not the packages required ASP.NET Core targets, which are present on the target system.
+
```

Comment:
I think @DamianEdwards slightly misspoke when he said the runtime store is added during publishing (4:40). Note how he restates it with "infer" immediately after that remark ... i.e., the packages of the runtime store are present (implicitly) when the app is published with the manifest (5:02-5:12) ... and in particular when the `All` metapackage is referenced by the app.

However ... the original language used in this topic is also incorrect (or perhaps just incomplete) imo ...

> The targets in [`Microsoft.NET.Sdk.Web`](https://github.com/aspnet/websdk) include manifests referencing the implicit package store that's available when the .NET Core SDK is installed on the target system.

Those targets may or may not rely on certain packages being present in the runtime store, but it isn't possible that they *require the runtime store implicitly*, which is inferred by the passage ... otherwise SCD deployments simply wouldn't work at all. I'm going to mod the language, and @DamianEdwards can say (as soon as I get the next commit posted) where the language needs additional work. Stand-by for another version of this topic ... I'm about to shoot it in now.